contentcuration/contentcuration/utils/assessment/qti/mathml/core.py

```
@@ -0,0 +1,206 @@
+
+class Munderover(MathMLScriptElement):
+    accent: Optional[bool] = None
+    accentunder: Optional[bool] = None
+    children: PresentationContentListLength3 = Field(default_factory=list)
+
+
+class Mprescripts(MathMLElement):
+    pass
+
+
+class Mmultiscripts(MathMLScriptElement):
```

Pylint (3.3.7)

[refactor] Too many ancestors (8/7)

(R0901)